emukit/examples/gp_bayesian_optimization/single_objective_bayesian_optimization.py

```diff
@@ -89,7 +90,7 @@ def _acquisition_chooser(self):
 
     def suggest_new_locations(self):
         """ Returns one or a batch of locations without evaluating the objective """
-        return self.candidate_point_calculator.compute_next_points(self.loop_state)[0]
+        return self.candidate_point_calculator.compute_next_points(self.loop_state)
```

Comment:
This change fails the tests. Why do you think it is needed?

Reply:
Hi Andrei,

Yes, I believe this change is needed. As the function name suggested (`compute_next_points`), the return can be multiple points. Therefore the return array should be of shape (n_points, n_dim).

I've fixed the test to check the return array shape.

Reply:
Makes sense, thanks for clarifying!